org.eclipse.jdt.core/model/org/eclipse/jdt/core/IJavaElementDelta.java

```diff
@@ -315,6 +315,14 @@ public interface IJavaElementDelta {
 	 */
 	public int F_ANNOTATIONS = 0x400000;
 
+	/**
+	 * Change flag indicating that the attributes of the element have changed.
+	 * Use {@link #getAttributeDeltas()} to get the added/removed/changed attributes.
```

Comment:
```suggestion
	 * Use {@link #getAttributeDeltas()} to get the added/removed/changed attributes.
	 * This flag is only valid if the element is an {@link IPackageFragmentRoot}.
	 *
	 * @see IClasspathAttribute
	 * @see IClasspathAttributeDelta
```

Reply:
`This flag is only valid if the element is an {@link IPackageFragmentRoot}.`

1. What "flag"?
2. I don't see how this belongs to a JavaDoc. Currently we only fill it for package fragment roots, but what is to prevent future code to add attributes for another type of element? Or am I missing something here?